packages/flutter_tools/templates/app/linux.tmpl/my_application.cc.tmpl

```diff
@@ -31,16 +33,19 @@ static void my_application_activate(GApplication* application) {
   fl_register_plugins(FL_PLUGIN_REGISTRY(view));
 
   gtk_widget_grab_focus(GTK_WIDGET(view));
-}
 
-static void my_application_class_init(MyApplicationClass* klass) {
-  G_APPLICATION_CLASS(klass)->activate = my_application_activate;
+  return true;
```

Comment:
Why `true` if the return type is `int`?

Reply:
Got confused between `local_command_line` and the signal handler for `command-line`. One returns an int, the other returns a bool :/